.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.3
```

Comment:
[CI] is still configured to run tests against ruby 2.7

[CI]: https://github.com/ezcater/ezcater_rubocop/blob/49f36ff06d6ac9dd2fdc539bfb519c82fc7f374f/.circleci/config.yml#L7

Reply:
CircleCI is not being invoked on pushes. I assume that is how the specs stopped working and wasn't noticed. Should this be moved to building via Jenkins, or do we still have a CircleCI subscription? I'm happy to do either.

Reply:
😬  it looks like it's running, but it's not reporting the status back for some reason.

https://app.circleci.com/pipelines/github/ezcater/ezcater_rubocop